packages/structure/src/index.ts

```diff
@@ -22,9 +22,18 @@ export async function printDiagnostics(
   const project = getProject(projectRoot)
   const formatOpts = { cwd: projectRoot, ...opts }
   try {
+    let exceptions = 0
     for (const d of await project.collectDiagnostics()) {
       const str = ExtendedDiagnostic_format(d, formatOpts)
       console.log(str)
+      // counts number of warnings (2) and errors (1) encountered
+      if (d.diagnostic.severity === 2 || d.diagnostic.severity === 1) {
```

Comment:
I wasn't able to use the ` DiagnosticSeverity.Warning ` and `DiagnosticSeverity.Error` syntax for some reason when importing `import { DiagnosticSeverity } from 'vscode-languageserver-types'` or using the export. It resolved in `runtime error: Cannot read property 'Error' of undefined`. Any insight into how to resolve that error would be helpful so that I can avoid using the severity numbers directly instead of a constant.

Reply:
Hmmm... Strange!

I can't think of a reason why those would be undefined, since it seems like we're using that import to set the diagnostic severity.

Reply:
^^ @aldonline

Reply:
I think it's probably OK that this isn't working as expected. Let's merge this!

Reply:
Thanks for the approval!